### DISCUSSION TITLE
OKD Serverless

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#856] [Q&A]
Hi.
Is it possible to install Openshift Serverless operator on OKD? This link in documentation is not working

![6077BCB4-2643-47A1-A2F2-788D70B9BCE8](https://user-images.githubusercontent.com/5565897/132133046-7e040dc7-af4f-48d6-b2e6-8bbb471fe3e1.jpeg)

**Comment by dkoci** [comment]
Anyone? 🙂

**Comment by vrutkovs** [comment]
See https://github.com/openshift-knative/serverless-operator/issues/375
